### PR TITLE
Add AMBER (Secret SNIP-20 token) to frontier

### PR DESCRIPTION
**Pull request by kent-3**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -167,6 +167,16 @@ const chainInfos = (
           coinGeckoId: "alter",
           coinImageUrl: "/tokens/alter.svg",
         },
+        {
+          type: "cw20",
+          contractAddress: "secret1s09x2xvfd2lp2skgzm29w2xtena7s8fq98v852",
+          coinDenom: "ALTER",
```
Comment (Sdpierret) [inline packages/web/config/chain-infos.ts:173]: coinDenom must match "AMBER" and not "ALTER"

**Review by Sdpierret — CHANGES_REQUESTED** [review]

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (kent-3) [reply packages/web/config/chain-infos.ts]: fixed, thanks

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -167,6 +167,16 @@ const chainInfos = (
           coinGeckoId: "alter",
           coinImageUrl: "/tokens/alter.svg",
         },
+        {
+          type: "cw20",
+          contractAddress: "secret1s09x2xvfd2lp2skgzm29w2xtena7s8fq98v852",
+          coinDenom: "AMBER",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:173]: Can you please register this to the chain registry?

File: packages/web/config/ibc-assets.ts [hunk 3]
```diff
@@ -1061,6 +1061,15 @@ export const IBCAssetInfos: (IBCAsset & {
     depositUrlOverride: "https://wrap.scrt.network",
     ics20ContractAddress: "secret1tqmms5awftpuhalcv5h5mg76fa0tkdz4jv9ex4",
   },
+  {
+    // AMBER
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:1065]: Please move to bottom of list

File: packages/web/config/chain-infos.ts (hunk 2 above)
Comment (kent-3) [reply packages/web/config/chain-infos.ts:173]: Yes I have submitted the pull request here: https://github.com/cosmos/chain-registry/pull/1094

**Pull request merged**